bitmovin.php

```diff
@@ -356,13 +399,42 @@ function generate_player($id)
 
     $html .= "conf.key = '" . $playerKey . "';\n";
 
-    $html .= "conf.source = conf.source || {}\n";
-    $html .= "conf.source.dash = '" . $dash . "';\n";
-    $html .= "conf.source.hls = '" . $hls . "';\n";
-    $html .= "conf.source.progressive = '" . $prog . "';\n";
-    $html .= "conf.source.poster = '" . $poster . "';\n";
+    $html .= "source = conf.source || {};\n";
+    $html .= "source.dash = '" . $dash . "';\n";
+    $html .= "source.hls = '" . $hls . "';\n";
+    $html .= "source.progressive = '" . $prog . "';\n";
+    $html .= "source.poster = '" . $poster . "';\n";
+
+    $analyticsKey = json_decode(get_post_meta($id, '_config_analytics_key', true));
+    $includeAnalytics = json_decode(get_post_meta($id, "_analytics_enabled", true)) == "on";
+
+    if($includeAnalytics){
+        $analytics_link = "https://cdn.bitmovin.com/analytics/web/2.0/bitmovinanalytics.min.js";
+        wp_register_script('bitmovin_analytics', $analytics_link);
+        wp_enqueue_script('bitmovin_analytics');
+        $custom_analytics = json_decode(get_post_meta($id, "_config_analytics_custom_conf", true));
+        if ($custom_analytics != "")
+        {
+            $html .= "conf.analytics = {" . $custom_analytics . "};\n";
+        }
+        $html .= "conf.analytics = conf.analytics || {};\n";
+        $html .= "conf.analytics.key = '" . $analyticsKey . "';\n";
+    }
+
+    if($isOldPlayer){
+        $html .= "conf.source = source;\n";
+        $html .= "var player = bitmovin.player(\"bitmovin-player\");\n";
+        if($includeAnalytics == "on"){
+            $html .= "bitmovin.analytics.augment(player);\n";

```

Comment:
With v2 of the collector the correct usage is simply:

`new bitmovin.analytics.adapters.Bitmovin7Adapter(player);`